pkg/terminalui/terminalui.go

```diff
@@ -0,0 +1,70 @@
+package terminalui
```

Comment:
Can you rename the package (aka folder) to `terminal`?
```suggestion
package terminal
```

Reply:
Yes sure